pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Dart bindings for Mozilla's PDF.js library
 homepage: https://github.com/Workiva/pdfjs_dart
 
 environment:
-  sdk: ">=2.11.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  quiver: ">=0.25.0 <3.0.0"
\ No newline at end of file
+  quiver: ">=3.0.1+1 <4.0.0"
```

Comment:
Running into issues with using > 3.0 for this, when consuming in something like markup_client. wdesk_sdk is still pinned below 3.0. I forget why we had to rev this again?

Reply:
Ah makes sense - quiver 3+ is null-safe

Seems like we could drop it as a dependency though since I think we're just making assertions with it

Reply:
We're currently bumping up compatibility for quiver to <4.0.0 everywhere, so this PR should be able to be picked back up.